zeppelin-interpreter-integration/pom.xml

```diff
@@ -203,4 +173,105 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>hadoop2</id>
+
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+
+      <properties>
+        <hadoop.version>${hadoop2.7.version}</hadoop.version>
+        <curator.version>2.7.1</curator.version>
+        <commons-io.version>2.4</commons-io.version>
```

Comment:
https://search.maven.org/artifact/commons-io/commons-io/2.11.0/jar is the latest one.

Reply:
Fixed